.github/workflows/scripts/start-rancher.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+flip() {
+    echo $1
+    exit 1
+}
+
+kubectl get ns | grep -s cattle && flip 'rancher already installed?'
+
+set -exu
+
+TMP="${TMP:-/tmp}"
+
+# `helm repo list` emits error code 1 when there are no repos to list
+set +e
+helm repo add cert-manager https://charts.jetstack.io
+helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+helm repo add jetstack https://charts.jetstack.io
+set -e
+
+helm repo update
+
+helm upgrade --install cert-manager --namespace cert-manager cert-manager/cert-manager --set installCRDs=true --set "extraArgs[0]=--enable-certificate-owner-ref=true" --create-namespace --wait --timeout=10m
+
+# kubectl get pods --namespace cert-manager
+kubectl rollout status --namespace cert-manager deploy/cert-manager --timeout 1m
+
+# Chart based
+
+CHART_PATH="${CHART_PATH:-/tmp/rancher.tgz}"
+helm upgrade --install rancher $CHART_PATH --namespace cattle-system --set hostname=localhost --wait --timeout=10m --create-namespace --version "${VERSION:-2.9}" --set rancherImage=rancher/rancher --set rancherImageTag="${RANCHER_IMAGE_TAG:-v2.9.0-alpha2}"
```

Comment:
Personally, I'd rather not have default values for these variables: `VERSION`, `RANCHER_IMAGE_TAG` because:
1. it's yet another place we probably want to update whenever we bump the rancher version 
2. If we don't update / forget to update the defaults, we could be running the integration test on an older version of rancher by mistake, not realizing it, and getting the CI to be green.